routes/api.php

```diff
@@ -73,6 +73,7 @@
         Route::put('rules', 'LegacyApiController@add_edit_rule')->name('edit_rule');
         Route::delete('rules/{id}', 'LegacyApiController@delete_rule')->name('delete_rule');
         Route::post('services/{hostname}', 'LegacyApiController@add_service_for_host')->name('add_service_for_host');
+        Route::get('oxidized/configsearch/{searchstring}', 'LegacyApiController@search_oxidized')->name('search_oxidized');
```

Comment:
Maybe change this to `/config/search/{searchstring}`? We can then use `/config` to return the devices config from the api at a later stage.

Reply:
Just updated it to follow your example.

Reply:
Thanks for updating the doc @laf  stupid of me to miss that :(